VSDiagnostics/VSDiagnostics/VSDiagnostics/Diagnostics/Attributes/FlagsEnumValuesAreNotPowersOfTwo/FlagsEnumValuesAreNotPowersOfTwoAnalyzer.cs

```diff
@@ -67,7 +79,7 @@ private void AnalyzeSymbol(SyntaxNodeAnalysisContext context)
 
             var enumName = context.SemanticModel.GetDeclaredSymbol(declarationExpression).Name;
             var enumMemberDeclarations =
-                declarationExpression.ChildNodes().OfType<EnumMemberDeclarationSyntax>().ToList();
+                declarationExpression.ChildNodes().SyntaxNodeOfType<EnumMemberDeclarationSyntax>(SyntaxKind.EnumMemberDeclaration);
```

Comment:
I'm pretty sure I made it generic so I would get an explicit type back, rather than just `SyntaxNode`.  If I change it, I'm going to need a bunch of casts in the code.